src/main/java/core/mate/academy/service/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> machines = new ArrayList<>();
+        machines.add(new Bulldozer());
+        machines.add(new Bulldozer());

```

Comment:
You have to create a constructor in Bulldozer (and in all models) with params from `Machine` (and then use `super(....)`) and unique params for each machine.